.markdownlint.yaml

```diff
@@ -0,0 +1,15 @@
+# For all the available configuration options, please view the following file:
```

Comment:
config file for markdownlint-cli tool. Unfortunately, pyproject.toml is not supported yet.

Reply:
FWIW: neuro-san uses the pip "readme_renderer" package to be sure stuff can be uploaded to pypi successfully.

Also: Does it have to be yaml? (sigh)

Reply:
I think it supports json and toml too. Its just that you need a separate file and cannot use pyproject.toml